backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -98,7 +98,7 @@ func TestCreateExperiment_Unauthorized(t *testing.T) {
 func TestCreateExperiment_Multiuser(t *testing.T) {
 	viper.Set(common.MultiUserMode, "true")
 	defer viper.Set(common.MultiUserMode, "false")
-	md := metadata.New(map[string]string{common.GoogleIAPUserIdentityHeader: "accounts.google.com:user@google.com"})
+	md := metadata.New(map[string]string{common.GetKubeflowUserIDHeader(): common.GetKubeflowUserIDPrefix() + "user@google.com"})
```

Comment:
nit: I'd prefer using constants in tests. Otherwise If something goes wrong We cannot know

Reply:
*If sth goes wrong with the getter function

Reply:
Done